hack/windows/Reset-Node.ps1

```diff
@@ -4,9 +4,9 @@ Performs a best effort revert of changes made to this host by 'kubeadm join'
 #>
 Write-Host "Running kubeadm reset"
 kubeadm.exe reset -f
-rm -r -f C:\etc\kubernetes\pki
-rm -r -fo C:\etc\kubernetes\pki
-rm -r -fo C:\var\lib\kubelet
+rm -r -force C:\etc\kubernetes\pki
+rm -r -force C:\etc\kubernetes\pki
```

Comment:
It seems L7 and L8 are the same.

Reply:
Thanks @lzhecheng, I remove the extra one